lsp/pylyzer.lua

```diff
@@ -22,4 +28,7 @@ return {
       checkOnType = false,
     },
   },
+  on_new_config = function(new_config, _)
+    new_config.cmd_env.ERG_PATH = vim.fs.joinpath(vim.uv.os_homedir(), '/.erg')
```

Comment:
don't need the slash if you are using joinpath()
```suggestion
    new_config.cmd_env.ERG_PATH = vim.fs.joinpath(vim.uv.os_homedir(), '.erg')
```